_helpers/meta-template/README.md

```diff
@@ -6,7 +6,7 @@
 
 ### Environment variables
 
-This project requires some environment variables to be set. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
+This project requires some environment variables to be set. A file named `.env` is used to store the values for those environment variables. New templates will only have a `.env` file, but it's possible to create a `.env.example` full of defaults and example values, suitable for committing to source control. On deployment, if `.env` does not exist, the values from `.env.example` will be used instead. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
```

Comment:
This kind of confusing to read. What does it mean that 
>New templates will only have a `.env` file 
shouldn't it be the reverse? By that I would expect new apps moving forward should be creating `.env.example` files

Isn't this also reversed? 
>On deployment, if `.env` does not exist, the values from `.env.example` will be used instead.

Reply:
For this specific section I'd only refer to `.env` because as Phil mentioned we don't actually create a `.env.example` file for *customers* and this file is customer facing. This change is purely an improvement for contributors to the project to have `.env` files during template development without accidentally committing their secrets.

Reply:
Good points! Since this is customer-facing and customers will likely not encounter `.env.example` files, I agree that not mentioning `.env.example` in this file is for the best.